src/patterns/dosdp-patterns/boneLengthAlongAxis.yaml

```diff
@@ -0,0 +1,103 @@
+---
+pattern_name: boneLengthAlongAxis
+
+pattern_iri: http://purl.obolibrary.org/obo/oba/patterns/boneLengthAlongAxis.yaml
+
+description: "A bone length trait that is the line along an axis of a bone."
+
+#  examples:
+#    - http://purl.obolibrary.org/obo/FOVT_0001308  # canine tooth medial-lateral diameter
+#    - http://purl.obolibrary.org/obo/FOVT_0001080  # calcaneous proximal breadth (synonym: calcaneous proximal length medial-lateral)
+
+contributors:
+  - https://orcid.org/0000-0002-7073-9172  # David Osumi-Sutherland
+  - https://orcid.org/0000-0001-8314-2140  # Ray Stefancsik
+  - https://orcid.org/0000-0002-7356-1779  # Nicolas Matentzoglu
+  - https://orcid.org/0000-0003-2699-3066  # Meghan A. Balk
+
+classes:
+  skeletal_element: UBERON:0004765
+  length: PATO:0000122
+  anatomical_axis: BSPO:0000010
+  NCBITaxon: OBI:0100026
+
+relations:
+  characteristic_of: RO:0000052
+  part_of: BFO:0000050
+
+annotationProperties:
+  exact_synonym: oio:hasExactSynonym
+  related_synonym: oio:hasRelatedSynonym
+  broad_synonym: oio:hasBroadSynonym
+  xref: oio:hasDbXref
+  measured_in_taxon: xsd:string
+  not_measured_in_taxon: xsd:string
+  comment: rdfs:comment
+
+data_list_vars:
+  def_ref: "'xref'"  # reference(s) to the source(s) of definition
+
+vars:
+  bone: 'skeletal_element'
+  axis: 'anatomical_axis'
+  fovt: "xsd:string"  # FOVT id
+  def_ref: "xsd:string"
+  taxa: "NCBITaxon"
+  your_comments: "xsd:string"  # comments, clarifications for users
+
+name:
+  text: "%s %s length"
+  vars:
+    - bone
+    - axis
+
+annotations:
+  - annotationProperty: related_synonym
+    text: "%s length %s"
+    vars:
+      - bone
+      - axis
+  - annotationProperty: broad_synonym
+    text: "%s length %s"
+    vars:
+      - bone
+      - axis
+  - annotationProperty: exact_synonym
+    text: "%s length %s"
+    vars:
+      - bone
+      - axis
+  - annotationProperty: xref
+    text: "FOVT:%s"  # give FOVT id here
+    vars:
+      - fovt
+  - annotationProperty: measured_in_taxon
+    text: "%s"
+    vars:
+      - taxa
+  - annotationProperty: not_measured_in_taxon
+    text: "%s"
+    vars:
+      - taxa
+
+  - annotationProperty: comment
+    text: "%s"
+    vars:
+      - your_comments
+
+def:
+  text: "The length of a %s along the %s."
+  vars:
+    - bone
+    - axis
+  annotations:
+    - annotationProperty: xref
+      value: def_ref
+
+logical_axioms:
+  - axiom_type: subClassOf
```

Comment:
If we do this, note that these terms will never automatically classify. They will be one massive flat list under `length`. Is this really what we want?

Reply:
no, I changed it to be an EQ.